.github/workflows/lint.yml

```diff
@@ -45,18 +48,25 @@ jobs:
             experimental/node_modules
             */*/node_modules
             experimental/*/*/node_modules
-          key: lint-${{ matrix.package-group }}-${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('**/package.json') }}-20211009.2
+          key: lint-${{ matrix.package-group }}-${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('**/package.json') }}-20211216.2
 
       # On a cache miss, install dependencies
       - name: Bootstrap
         if: steps.cache.outputs.cache-hit != 'true'
         working-directory: ${{ matrix.package-group }}
         run: |
           npm install --ignore-scripts
-          npx lerna bootstrap --no-ci --ignore-scripts --hoist --nohoist='zone.js' -- --only=dev
+          npx lerna bootstrap --no-ci --hoist --nohoist='zone.js'
 
       - name: Lint
         working-directory: ${{ matrix.package-group }}
         run: |
           npm run lint
           npm run lint:examples
+
+      - name: Lint doc files
+        if: ${{ matrix.package-group == '.' }}
+        run: |
+          npm run compile
```

Comment:
Is it required to compile in order to check links?

Reply:
AFAICT, typedoc didn't work with a full compilation with "Warning: Could not determine the entry point for "/Volumes/Y/Developer/notebook/opentelemetry-js/packages/opentelemetry-core/package.json". Package will be ignored." for all packages on my local clean checkout.

Reply:
ok i remember now i had the same thing when i was fixing the typedoc config for doc publishing